src/server/game/Handlers/GroupHandler.cpp

```diff
@@ -911,7 +911,7 @@ void WorldSession::HandleRequestPartyMemberStatsOpcode(WorldPacket &recvData)
     recvData >> Guid;
 
     Player* player = ObjectAccessor::FindConnectedPlayer(Guid);
-    if (!player)
+    if (!player || !GetPlayer()->GetGroup() || !(player->GetGroup()->GetGUID() == GetPlayer()->GetGroup()->GetGUID()))
```

Comment:
You need to check if player->GetGroup() is null also

Reply:
> You need to check if player->GetGroup() is null also.

Corrected.

> You can compare the result of GetGroup() itself, no need to compare the guid, unless you have reasons that I am not aware about to do so.

Remnant from trying to get the condition to pass when I didn't realize I had a double negative hanging around. Corrected as well.

Reply:
One could remove 1 of the null checks now that we check for != but we keep it like that.